### PR TITLE
project copy now reflects active projects

### LINKED ISSUE #250: Update Project Copy to reflect oneleif active projects
Currently have a lot of dummy copy to take up space, need to make this view reflect the actual projects of oneleif

### PR DESCRIPTION
**Pull request by roramigator**
# summary

## result

the proper files have been updated with the active projects, the path `/projects` now renders the new data: 

Screen Shot 2021-01-22 at 8 17 11 PM

## concerns

* there were no images provided for each individual project, the default image (**A**) is been used as a placeholder.
* project name (**B**) is align to the left if there's a line break.
* the project description (**C**) was taken from repos and websites.

Screen Shot 2021-01-22 at 8 18 40 PM

Fixes #250

### CODE REVIEW

**Review by 0xLeif — APPROVED** [review]
LGTM 👍

**Comment by will-nemo** [comment]
To fix the align left when there is a line break you can just add text-align: center to the styling on the card. That should fix the issue

File: src/components/Footer/js/footerLinks.js [hunk 1]
```diff
@@ -1,6 +1,62 @@
...
+// ]
+
+/************************************
+ * Proposal
+ ************************************/
+// the data can be generated creating the array of objects
+// pros: less code
+// cons: logic implementation if path and internal differ between objects
+
+const parseLinks = projectCards.map(project => {
```
Comment (will-nemo) [inline src/components/Footer/js/footerLinks.js:5]: true at this point i like this idea, we can have it randomly select 3 projects each time .You should submit an issue to add that functionality. At some point we'd want it to load those links dynamically off the backend

**Review by will-nemo — CHANGES_REQUESTED** [review]
Couple changes but should be good after that

File: src/components/Footer/js/footerLinks.js (hunk 1 above)
Comment (roramigator) [reply src/components/Footer/js/footerLinks.js:5]: understood.

**Review by will-nemo — APPROVED** [review]
LGTM

**Pull request merged**